src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listEmployees = [...document.querySelectorAll('li')];
+
+const toNum = (el) => +el.match(/\d+/g).join('');
+
+const sortList = (list) => {
+  list.sort((a, b) => {
+    return toNum(b.dataset.salary)
+      - toNum(a.dataset.salary);
+  });
+
+  document.querySelector('ul').append(...list);
+};
+
+const getEmployees = (list) => {
+  const newEmployees = [];
+
+  for (let i = 0; i < list.length; i++) {
+    const employee = {};
+
+    employee.name = list[i].innerText;
+    employee.position = list[i].dataset.position;
+    employee.salary = list[i].dataset.salary;
+    employee.age = list[i].dataset.age;
+
+    newEmployees.push(employee);
+  }
```

Comment:
use `map `method. it can make your code shorter

Reply:
Thank you! Done!